source/sass/blocks/form-options.scss

```
@@ -44,13 +45,14 @@
     display: grid;
     grid-template-columns: 1fr 1fr;
     grid-template-rows: 1fr 1fr;
-    column-gap: 64px;
-    row-gap: 32px;
+    column-gap: 55px;
+    row-gap: 20px;
```

А чтобы нормально работал и список тоже, стоит задать ему ширину 
```suggestion
    row-gap: 20px;
    width: 100%;
```